### ISSUE TITLE
Error while brew install unused "Failed to load interface for..."

### ISSUE THREAD

**Issue by wRadion** [joshuaclayton/unused#80]
Hi,

I'm trying to install unused with brew : `brew install unused`
But I've got an error :

```
src/Unused/Projection/Transform.hs:10:1: error:
    Failed to load interface for ‘Text.Inflections.Parse.Types’
    Perhaps you meant
      Text.Inflections.Types
      Text.Inflections.Parse.CamelCase
      Text.Inflections.Parse.SnakeCase
    Use -v to see a list of the files searched for.
cabal: Leaving directory '.'
cabal: Error: some packages failed to install:
unused-0.6.1.0 failed during the building phase. The exception was:
ExitFailure 1
```

Can someone help me?

**Comment by joshuaclayton** [comment]
@wRadion thanks for reaching out, and apologies for the confusion here!

I've pushed up a fix I've confirmed works locally - can you rerun the installation with homebrew's `--HEAD` flag and let me know if that fixes it for you?

**Comment by wRadion** [comment]
@joshuaclayton it worked! Thanks a lot for the fast response and fix :).

**Comment by joshuaclayton** [comment]
@wRadion excellent, glad to hear! I've released [0.7.0.0](https://github.com/joshuaclayton/unused/releases/tag/v0.7.0.0) and [bumped the Homebrew formulae](https://github.com/joshuaclayton/homebrew-formulae/commit/d517938aba55cd281cf02f9647f3ad55306192a6) so others don't run into the same issues.

Thanks again for reporting!
